resources/serverless/templates/cluster-role.yaml

```diff
@@ -5,13 +5,61 @@ metadata:
   labels:
     {{- include "tplValue" ( dict "value" .Values.global.commonLabels "context" . ) | nindent 4 }}
 rules:
+- apiGroups:
```

Comment:
Duplication of configmap.

Reply:
It's because I've fixed rbacs generated by `controllers/kubernetes`, and now they also generate an entry here, but K8s should be able to "squash" all of those, so that there are no duplicates. From it's perspective it's just one simple operation, and from ours it is beneficial in a sense that each controller ensures in has proper rbacs set - imagine a situation in which you remove one of the controllers we now have, removing also marker which generates the rbacs -> in turn generated rbacs could break.

One idea I have is to add `rbac.go` file with `// +build rbac` comment (so it's not complited), and put ALL rbac markers in there, preventing duplication through the codebase

or even somewhere in `main.go`